wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,458 @@
+# Mapping and modding timeline
+
+Mapping and modding systems are constantly improving. The **Mapping and modding timeline** aims to document the biggest system changes and to serve as a reference tool when incorporating new changes.
```

Comment:
I may be wrong, but "The timeline aims to serve as a reference when incorporating new changes" makes it sound like the timeline itself would make the changes.